.gitlab/deploy_containers/deploy_containers_a7.yml

```diff
@@ -82,6 +82,23 @@ deploy_containers-a7_internal-rc:
     VERSION: 7-rc
 
 
+deploy_containers-ot:
+  extends: .docker_publish_job_definition
+  stage: deploy_containers
+  rules:
+    !reference [.manual]
+  dependencies: []
+  before_script:
+    - source /root/.bashrc
+    - export VERSION="$(inv agent.version --major-version 7 --url-safe --pipeline-id $PARENT_PIPELINE_ID)"
+  parallel:
+    matrix:
+      - IMG_SOURCES: ${SRC_AGENT}:v${PARENT_PIPELINE_ID}-${CI_COMMIT_SHORT_SHA}-7-ot-amd64,${SRC_AGENT}:v${PARENT_PIPELINE_ID}-${CI_COMMIT_SHORT_SHA}-7-ot-arm64
+        IMG_DESTINATIONS: ${AGENT_REPOSITORY}:${VERSION}-ot-beta
+      - IMG_SOURCES: ${SRC_AGENT}:v${PARENT_PIPELINE_ID}-${CI_COMMIT_SHORT_SHA}-7-ot-jmx-amd64,${SRC_AGENT}:v${PARENT_PIPELINE_ID}-${CI_COMMIT_SHORT_SHA}-7-ot-jmx-arm64
+        IMG_DESTINATIONS: ${AGENT_REPOSITORY}:${VERSION}-ot-beta-jmx
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution. -->
**❓ question**
Have you considered using a dedicated repo? With registry it should be enough to use a different name

Reply:
I don't think we want a dedicated repo; we'd rather be consistent with how we're actually going to deliver this which will be in the `agent` repo with a specific tag. If this makes life very hard for you folks, we can use a separate repo. But overall, my reasoning was about us wanting the the expectation to be that the opentelemetry build is just a variant of the Datadog Agent, and thus better suited for a tag.